packages/cli/src/GsnTestEnvironment.ts

```diff
@@ -176,6 +176,7 @@ class GsnTestEnvironmentClass {
     const contractInteractor = new ContractInteractor(
       {
         provider: new Web3.providers.HttpProvider(host),
```

Comment:
better make `providers[]` parameter, which is an array of all underlying providers. 
the first one is the "primary", and the rest are "alternates"